edx_when/api.py

```diff
@@ -151,7 +151,7 @@ def _processed_results_cache_key(
     if allow_relative_dates:
         cache_key += '.with-rel-dates'
     if subsection_and_higher_only:
-        cache_key += '.subsection_and_higher_only'
+        cache_key += '.subsection_and_higher_only_v2'
```

Comment:
Oy. Probably not worth a comment, but an odd line for some future engineer.

Reply:
Odd lines are worth commenting. I'll add something, ty.